.github/workflows/ci.yaml

```diff
@@ -12,21 +12,23 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-name: Neo4j CI
+name: ci
 
 on:
-  workflow_dispatch:  # testing only, trigger manually to test it works
+  workflow_dispatch: # testing only, trigger manually to test it works
   push:
-    branches: [main]
+    branches:
+      - main
   pull_request:
-    branches: [main]
-    types: [opened, synchronize]
-  pull_request_review:
-    types: [submitted]
+    branches:
+      - main
+    types:
+      - opened
+      - synchronize
+      - reopened
 
 jobs:
   build:
-    name: CI using Neo4j
```

Comment:
minor nit: I would keep a name here, at least for completness.

Reply:
Addressed ✅